documentation/src/main/asciidoc/topics/ref_hotrod_countermanager.adoc

```diff
@@ -0,0 +1,14 @@
+[id='remotecache_countermanager-{context}']
+= Remotely Using Counters
```

Comment:
The title seems off. Previous sections are "Basic API", "Streaming API" and so on... shouldn't be "Counter API"? or similar?